src/Avalonia.Build.Tasks/Avalonia.Build.Tasks.csproj

```diff
@@ -90,9 +90,30 @@
       <Compile Include="../Avalonia.Controls/GridLength.cs">
         <Link>Markup/%(RecursiveDir)%(FileName)%(Extension)</Link>
       </Compile>
+      <Compile Include="../Avalonia.Visuals/RelativePoint.cs">
+        <Link>Markup/%(RecursiveDir)%(FileName)%(Extension)</Link>
+      </Compile>
+      <Compile Include="../Avalonia.Visuals/Media/Brush.cs">
```

Comment:
That's quite a lot of files and quite a bit of changes that could be replaced by just parsing the color.

Reply:
Should it be added directly to Avalonia.Build.Tasks project?

Reply:
I think brush-related stuff isn't needed by compiler at all. We only support converting solid color brushes, so just using existing color parser is enough.

Reply:
But `KnownColors.ToBrush(Avalonia.Media.KnownColor.Blue)` looks better than `new BrushConverter().ConvertFrom((ITypeDescriptorContext)(object)context, CultureInfo.InvariantCulture, "Blue")`

Reply:
I think idea here is to parse a color to an integer and emit `new SolidColorBrush(parsedColor)` instead. Which does not require referencing new files. We could even consider emitting immutable brushes (although that might not work too well for animations).